src/Native/Runtime/unix/UnwindHelpers.cpp

```diff
@@ -346,10 +363,13 @@ bool UnwindHelpers::StepFrame(REGDISPLAY *regs)
     uintptr_t pc = regs->GetIP();
 
     UnwindInfoSections uwInfoSections = LocateUnwindSections(pc);
+
+#if defined(_TARGET_AMD64_)
```

Comment:
It may be better to use the platform neutral defines for this, like `_LIBUNWIND_SUPPORT_DWARF_UNWIND`, etc.